packages/mui-joy/src/Tooltip/Tooltip.tsx

```diff
@@ -74,9 +73,6 @@ const TooltipRoot = styled('div', {
     ...(!ownerState.disableInteractive && {
       pointerEvents: 'auto',
     }),
```

Comment:
I think this should be flipped as well?

```js
…ownerState.disableInteractive && {
  pointerEvents: 'none'
}
```

Reply:
this pointer event is set when disable interactive is true. I think this one is right, since disableInteractive means that the tooltip will disappear after the mouse leave.